github/PullRequest.py

```diff
@@ -447,9 +447,9 @@ def create_review_comment(
             "RIGHT",
         ], start_side
         assert in_reply_to is github.GithubObject.NotSet or isinstance(in_reply_to, int), in_reply_to
-        assert subject_type is github.GithubObject.NotSet or subject_type in [
-            "LINE",
-            "FILE",
+        assert subject_type is github.GithubObject.NotSet or subject_type.lower() in [
```

Comment:
On a second thought, lets remove this `lower` so we encode the current known API requirement:
```suggestion
        assert subject_type is github.GithubObject.NotSet or subject_type in [
```
Once docs are updated, we can fix all string assertions.